lib/explorer/series.ex

```diff
@@ -476,6 +476,41 @@ defmodule Explorer.Series do
   @spec last(series :: Series.t()) :: any()
   def last(series), do: Shared.apply_impl(series, :last, [])
 
+  @doc """
+  Returns a series from two series, based on a predicate.
+
+  The resulting series is built by evaluating each element of
+  `pred` and returning either the corresponding element from
+  `on_true` or `on_false`.
+
+  `pred` must be a boolean series. `on_true` and `on_false` must be
+  a series of the same length as `pred`.
+  """
+  @doc type: :transformation
+  @spec select(predicate :: Series.t(), on_true :: Series.t(), on_false :: Series.t()) :: any()
```

Comment:
I believe we always return a `Series` here.

```suggestion
  @spec select(predicate :: Series.t(), on_true :: Series.t(), on_false :: Series.t()) :: Series.t()
```